status_app.py

```diff
@@ -205,6 +206,7 @@ def __init__(self, settings):
             ("/flowcells", FlowcellsHandler),
             ("/flowcells/([^/]*)$", FlowcellHandler),
             ("/flowcells_plot", FlowcellPlotHandler),
+            ("/data_delivered_plot", DeliveryPlotHandler),
             ("/flowcell_count_plot", FlowcellCountPlotHandler),
```

Comment:
I think you need to remove this one as well if you're removing the flowcell count plot. Also the import of this handler.